code/modules/admin/topic.dm

```diff
@@ -36,6 +43,7 @@
 
 		message_admins("[key_name_admin(usr)] marked [C.key]'s admin help as an IC issue.")
 		log_admin_private("[key_name(usr)] marked [C.key]'s admin help as an IC issue.")
+		spamcooldown = world.time + 300 // 30 seconds

```

Comment:
30 seconds cooldown of rejecting any ahelp(and not just the same) is just awful.